install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -538,15 +538,22 @@ data:
   enable-auto-protect-node-port-range: {{ .Values.nodePort.autoProtectPortRange | quote }}
 {{- end }}
 {{- if hasKey .Values "loadBalancer" }}
+{{- if .Values.loadBalancer.standalone }}
+  datapath-mode: lb-only
+{{- end }}
 {{- if hasKey .Values.loadBalancer "mode" }}
-  node-port-mode: {{ .Values.loadBalancer.mode | quote }}
+  bpf-lb-mode: {{ .Values.loadBalancer.mode | quote }}
 {{- end }}
 {{- if hasKey .Values.loadBalancer "algorithm" }}
   bpf-lb-algorithm: {{ .Values.loadBalancer.algorithm | quote }}
 {{- end }}
 {{- if hasKey .Values.loadBalancer "acceleration" }}
-  node-port-acceleration: {{ .Values.loadBalancer.acceleration | quote }}
```

Comment:
Are we deprecating this option? if yes it needs to be added in the upgrade notes

Reply:
No, please see https://github.com/cilium/cilium/commit/d73c572fcc970bcb19bbcb86373dcc3595ec00f7 linked in the git commit msg.